source/docs/appendices/migration-guides.md

```diff
@@ -7,6 +7,217 @@ This area will contain the Migration steps to follow for upgrading your store to
+
+```diff
+- intl.formatMessage("Hello <strong>Bob</strong>");
++ intl.formatMessage("Hello <strong>Bob</strong>", {
++   strong: (text) => <strong>{text}</strong>,
++ });
+```
+
+You must also ensure that your app doesn't use `<FormattedHTMLMessage>` or `intl.formatHTMLMessage` that have been removed. See [the list of breaking changes](https://formatjs.io/docs/react-intl/upgrade-guide-4x/#breaking-api-changes).
+
+### `formsy-react` updated to 2.x
+
+`formsy-react` has been updated from 1.1.5 to 2.0.3. Some internal methods have been
+renamed and subtle low-level behavior changes were introduced (required errors by default in errors
+etc…).
+
+**If you were using [Front-Commerce's wrappers introduced in 2.0.0-rc.0](https://developers.front-commerce.com/docs/appendices/migration-guides.html#Abstract-Formsy) you might not encounter any issues**.
+Otherwise, we recommend to refactor your code to use them!
+
+See [Formsy's upgrade guide](https://github.com/formsy/formsy-react#1x-to-2x-upgrade-guide) for exhaustive details about low-level changes.
+
+### Stripe embedded payment updated
+
+Stripe libraries were updated.
+
+The [`stripe`](https://www.npmjs.com/package/stripe) Node.js library was updated from 7.15.0 to 8.60.0. See [the migration guide](https://github.com/stripe/stripe-node/wiki/Migration-guide-for-v8) in case your application relies on advanced features.
+
+The react library has bigger changes. Front-Commerce is now using the official and rebranded [@stripe/react-stripe-js](https://www.npmjs.com/package/@stripe/react-stripe-js) instead of the deprecated [react-stripe-elements](https://github.com/stripe/react-stripe-elements) library. If you overrode Stripe components, please check [the "migrating from react-stripe-elements" guide](https://github.com/stripe/react-stripe-js/blob/master/docs/migrating.md) for migration instructions (imports etc…).
+
+We rewrote default components to use new Stripe components:
+- the `<StripeCheckoutElement>` has been simplified in FC, please reapply your changes to the latest version if you overrode it
+- the `<StripeCheckout>` component was totally rewritten to use hooks. It is very unlikely that you overrode it, but if you did please update your override.
+
+### Miscellaneous and unlikely impacting backwards incompatible changes
+
+Some minor changes were introduced in dependencies or while removing deprecated code. They are unlikely impacting a standard application, but worth mentioning in case you relied on some low-level APIs.
+
+- The `<ExistingAddress />` and `<NewAddress />` components used in the checkout were updated to use the `<Checkbox />` component as they were using a deprecated one. Make sure it doesn't have an impact on your styles.
```

Comment:
This also means that any one that overrode these two components will have a broken build because the component no longer exists. That's not unlikely to happen imo.

Reply:
Yes, not very likely but I'll add a few words to make it explicit.